Create 2022-06-21-ElixirAHM22.md (#993)
usegalaxy-eu/website: _posts/2022-06-21-ElixirAHM22.md

**ELIXIR All Hands 2022**

During 6-9th of June, members of the European Galaxy team travelled to Amsterdam to join the [ELIXIR All Hands meeting 2022](https://elixir-europe.org/all_hands_2022).

{% twitter https://twitter.com/BSerranoSolano/status/1533824511795142657?t=wIrbNbACZKjIUJC-HCUfzQ&s=09 %}

There was a broad presence of Galaxy all over the 3 days of the meeting across the different workshops (parallel sessions) and plenary talks and discussions. We will highlight here the participants.

> {% twitter https://twitter.com/denbiOffice/status/1534872847126089728 %}

Reviewer: or add a sentence at the beginning saying: we joined other colleagues from ELIXIR-DE...

On the first day; the 7th of June, Anup Kumar and Kaivan Kamali presented in the [AI-powered data analysis within the European Galaxy workshop](https://docs.google.com/document/d/17IVSvLmnianF6VOLO7GQfwPFDTpm9DF95ShHXnaarNo/edit#heading=h.8yacamuak08k) conducted using an [image classification tutorial](https://training.galaxyproject.org/training-material/topics/statistics/tutorials/CNN/tutorial.html). In the workshop, a convolutional neural network was trained to recognise handwritten digits using Galaxy Europe's deep learning tools in a stepwise manner. The steps included importing datasets, creating an architecture of the convolutional neural network, training it on imported train datasets and evaluating the trained model using test datasets and visualisation techniques. 

{% twitter https://twitter.com/BSerranoSolano/status/1534113948135747584 %}

> On that same day afternoon, the ELIXIR-Galaxy community organised a workshop on [Earth Biodiversity](https://docs.google.com/presentation/d/1YvJsPK0DAcl2_pFwfCcYx9KnEM0eRyMcI82uOVUCZVg/edit) conducted by Frederik Coppens (ELIXIR-BE) and Björn Grüning (ELIXIR-DE). There were many talks, 7 in total,  including Engy Nasr’s presentation [slides: 83-102]. The workshop topic is highly relevant at the moment thanks to initiatives like the [Vertebrate Genomes Project (VGP)](https://vertebrategenomesproject.org/) and [European Reference Genome Atlas (ERGA)](https://erga-online.eu/), which will sequence and assemble the genomes of thousands of species over the next few years, and the plans that have been outlined in a submitted Horizon Europe call.

Reviewer: ```suggestion
On that same afternoon, the ELIXIR-Galaxy community organised a workshop on [Earth Biodiversity](https://docs.google.com/presentation/d/1YvJsPK0DAcl2_pFwfCcYx9KnEM0eRyMcI82uOVUCZVg/edit). The workshop topic is highly relevant at the moment thanks to initiatives like the [Vertebrate Genomes Project (VGP)](https://vertebrategenomesproject.org/) and [European Reference Genome Atlas (ERGA)](https://erga-online.eu/), which will sequence and assemble the genomes of thousands of species over the next few years, and the plans that have been outlined in a submitted Horizon Europe call.
```

Author: that was in the abstract I will remove it

> Moreover, the new [microGalaxy community](https://galaxyproject.org/community/#communities-of-practice) were presented, and how all these initiatives align with the other ELIXIR Communities and Focus Groups (e.g. Biodiversity, Federated Human Data, Human Copy Number Variation, Microbiome, ...). That also included the latest updates about our SARS-CoV-2 efforts and how the experience gained can be transferred to other outbreaks and diseases.

Author: also from the abstract, shall I remove it as well ? in my presentation I have just mentioned microGalaxy community

Engy’s presentation was about microbiome data analysis with Galaxy: example of clouds. Her presentation's goal was to show the ability of the Galaxy platform with all its tools and workflows in analysing microbial data, leading the audience to where and how they can find and use them for their biome analysis. Moreover, the presentation also has guidance on the required training to use or contribute to Galaxy from the [Galaxy Training Network](https://training.galaxyproject.org/). That was shown by giving an example of one of the Galaxy team’s ongoing projects on cloud and aerosol metagenomics and metatranscriptomics datasets, where Engy briefly presented her work-in-progress workflow and results from analysing these datasets focusing more on biodiversities. 

{% twitter https://twitter.com/BSerranoSolano/status/1534163078165811203 %}

In the same workshop, Björn Grüning gave [updates from the Galaxy-verse](https://docs.google.com/presentation/d/1YvJsPK0DAcl2_pFwfCcYx9KnEM0eRyMcI82uOVUCZVg/edit#slide=id.g130105c60be_0_75).

> In parallel to this workshop, Bérénice Batut presented at the [Establishing a roadmap for the Communities towards a sustainable co-production model with the Training Platform workshop](https://docs.google.com/document/d/1vlHODVnN9RFTp2fr2NCcLV4ujb8Js3EH6NqXHfP_F9E/edit#). Her training topic was: [Brief outline of the participating communities with regard to Training](https://docs.google.com/presentation/d/10hMdBFX2x3pTc95HyEzG6xg0V0nKBtd0batf5M8SJYg/edit#slide=id.g12d07c72490_0_237). 

Reviewer 2: ```suggestion
In parallel to this workshop, Bérénice Batut presented the [outline of the  Galaxy Community with regards to Training](https://docs.google.com/presentation/d/10hMdBFX2x3pTc95HyEzG6xg0V0nKBtd0batf5M8SJYg/edit#slide=id.g1229a6b8fb8_0_50)
 at the [Establishing a roadmap for the Communities towards a sustainable co-production model with the Training Platform workshop](https://docs.google.com/document/d/1vlHODVnN9RFTp2fr2NCcLV4ujb8Js3EH6NqXHfP_F9E/edit#). She presented the efforts of the [Galaxy Training Network](https://training.galaxyproject.org/): the collection of >260 tutorials in 23 topics, the FAIRification of the training material, the work on developing learning paths, etc
```

> On the second day, the morning of the 8th of June, Nicola Soranzo presented in the ELIXIR Communities and Connectedness session. His topic was about [“the Galaxy community: “Gateway for integration across Communities” ](https://docs.google.com/presentation/d/1Ya-UDX5QuOVKEVQ1RuKZqdful4y425vh5qyap0QWoyU/edit?usp=sharing).

Reviewer: ```suggestion
On the second day, the morning of the 8th of June, Nicola Soranzo presented in the ELIXIR Communities and Connectedness session. His topic was about [“the Galaxy community: Gateway for integration across Communities”](https://docs.google.com/presentation/d/1Ya-UDX5QuOVKEVQ1RuKZqdful4y425vh5qyap0QWoyU/edit?usp=sharing).
```

Author: I will remove them all

On the afternoon of the 8th of June, Galaxy was also present in the poster session. Anup Kumar summarised his work on [An accessible artificial intelligence infrastructure using a docker-based Jupyterlab in Galaxy](https://drive.google.com/file/d/1a96OvZpNjLEFK8UqFrNgt7SOof5NFyA6/view?usp=sharing), which showcases a few recent scientific use cases of GPU-enabled Jupyterlab, being used for creating end-to-end artificial intelligence projects running entirely on Galaxy Europe.

Also in the afternoon of the same day, Beatriz Serrano-Solano participated in the [“COVID-19” workshop](https://docs.google.com/document/d/1FKBYBxehpaxFT_vLURwq5TbyQjxI4sYiSKeULT0ioso/edit), presenting the [work done by the Galaxy community](https://docs.google.com/presentation/d/1scNZjk61eCZzk3qrKos96zKcBoFFJP6BdqRZ302iT5M/edit#slide=id.g128cc83e3ba_1_4) since the beginning of the pandemic and highlighting the importance of Open Science over the last two years.

{% twitter https://twitter.com/CaroleAnneGoble/status/1534529389697937409 %}
{% twitter https://twitter.com/bjoerngruening/status/1534527617939939332?t=2z2d7QvvIUWp56aY0gtYBg&s=09 %}

> On the third and last day of the event, the 9th of June, several talks were given by Björn Grüning. In the morning, he presented at the [Enabling Science at European Scale session](https://docs.google.com/document/d/1uqF_AogQ2N4hLstlzH1QGsRoWE0IHKmeLMKVBkYfSb0/edit), in which he talked about the different ways how to scale Galaxy. Just before lunch, Björn and others also gave a short talk in the [Science driven improvements to tools ecosystems workshop](https://docs.google.com/document/d/1VA_9_1TOECPL26exs4Nyow_Y2V9RGzOnzF0WYcmLxyo/edit#heading=h.8yacamuak08k). In the afternoon, Björn has participated in two parallel workshops; the first was about [single cell](https://docs.google.com/document/d/13pOfy2nkajjzsjvlGV_ch_0EvB6say1LBdOKtbXyVsY/edit#) and the second was about [ELIXIR tools platform](https://docs.google.com/document/d/14XEkip1g0S8-eqySM7uZMCXFzPqjSQX-b-i1qxVwxrU/edit) where he talked about [BioConda](https://anaconda.org/bioconda) and [Biocontainers](https://biocontainers.pro/). 

Reviewer: ```suggestion
On the third and last day of the event, the 9th of June, several talks were given by Björn Grüning. In the morning, he presented at the [Enabling Science at European Scale session](https://docs.google.com/document/d/1uqF_AogQ2N4hLstlzH1QGsRoWE0IHKmeLMKVBkYfSb0/edit), in which he talked about the different ways how to scale Galaxy. Just before lunch, Björn and others also gave a short talk in the [Science driven improvements to tools ecosystems workshop](https://docs.google.com/document/d/1VA_9_1TOECPL26exs4Nyow_Y2V9RGzOnzF0WYcmLxyo/edit#heading=h.8yacamuak08k). In the afternoon, Björn participated in two parallel workshops; the first was about [single cell](https://docs.google.com/document/d/13pOfy2nkajjzsjvlGV_ch_0EvB6say1LBdOKtbXyVsY/edit#) and the second was about [ELIXIR tools platform](https://docs.google.com/document/d/14XEkip1g0S8-eqySM7uZMCXFzPqjSQX-b-i1qxVwxrU/edit) where he talked about [BioConda](https://anaconda.org/bioconda) and [Biocontainers](https://biocontainers.pro/). 
```

Reviewer 3: I could not be at the ELIXIR tools platform as this was overlapping with the Single-Cell one. In Single-cell I talked about workflowhub.

{% twitter https://twitter.com/ELIXIRnodeBE/status/1534807219740581888 %}

On the same day, Beatriz Serrano-Solano presented the [EOSC-Life consortium and the participation of the Galaxy community](https://docs.google.com/presentation/d/1Wv1teYk2Egww6fg5RbD5iq4s_5qkaVby2OXHmkaYVW4/edit#slide=id.g130fad382dc_0_0) in different aspects of the project, from the technical side (tools, workflows, infrastructure) to the use cases ([chemoinformatics, marine biology, imaging](https://galaxyproject.eu/posts/2021/03/10/eosc-life-d6/), reference data, [food safety](https://galaxyproject.eu/posts/2021/12/08/pathogen-detection-eosc-life-grant/) and [SARS-CoV-2](https://galaxyproject.eu/news?tag=COVID-19)), training and outreach. She also presented the Galaxy platform as the link between different EOSC projects, like EOSC-Nordic and EOSC-Pillar.

{% twitter https://twitter.com/bjoerngruening/status/1534908328010301444?t=om0kEUUm2OecL7kxKMlkBA&s=09 %}
{% twitter https://twitter.com/jtedds/status/1534906599260790784?t=6SJL2AQCTZiejDOFI3A0zw&s=09 %}
{% twitter https://twitter.com/denbiOffice/status/1534904940451311616?t=IZ02fu4TyMf5E68TUeTRgg&s=09 %}

> {% twitter https://twitter.com/EuroBioImaging/status/1534919493377466368?t=L7A-7_guaCo6kBoRNfXgMA&s=09 %}

Reviewer: What about adding a short paragraph to close the post? Examples: we were happy to attend, first in-person AH for some of us, great to see people again, we look forward to joining next year, Galaxy was widely mentioned beyond the talks, also part of discussions, thank all the people that presented...
